core/src/test/java/com/xmartlabs/base/core/observers/MainObserversWithErrorHandlingTest.java

```diff
@@ -0,0 +1,94 @@
+package com.xmartlabs.base.core.observers;
+
+import android.support.annotation.Nullable;
+
+import com.annimon.stream.Objects;
+import com.annimon.stream.Stream;
+
+import org.junit.Before;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.reactivex.functions.Consumer;
+import lombok.Data;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+import timber.log.Timber;
+
+abstract class MainObserversWithErrorHandlingTest {
+  static final String OBSERVABLE_ACTION_EXCEPTION = "Observable Action Exception";
```

Comment:
Maybe just using it in the place, and maybe avoiding inheritance. I don't know. Perhaps the repetition isn't worth it.